src/os/bluestore/ZonedAllocator.cc

```diff
@@ -31,7 +31,8 @@ ZonedAllocator::ZonedAllocator(CephContext* cct,
       // The last 32 bits of |block_size| is holding the actual block size.
       block_size((block_size & 0x00000000ffffffff)),
       zone_size(((block_size & 0x0000ffff00000000) >> 32) * 1024 * 1024),
-      starting_zone_num((block_size & 0xffff000000000000) >> 48),
+      first_seq_zone_num((block_size & 0xffff000000000000) >> 48),
```

Comment:
I agree. I wish Ceph had adopted Google convention so that I could just use block_size_ for members and block_size for parameters. Ceph prefers block_size_m for members, but it is not universally observed and changing just block_size to that will look ugly. I've renamed the parameter to blk_size. Let me know if you have a better idea.